js/lightbeam.js

```diff
@@ -1,9 +1,33 @@
 async function renderGraph() {
-  const canvas = document.getElementById('canvas');
-  const context = canvas.getContext('2d');
   const websites = await store.getAll();
 
-  viz.draw(context, websites);
+  // reformat data for D3
```

Comment:
Based on [D3 documentation](https://bost.ocks.org/mike/join/) and my own experimentation, D3 has built-in methods that calculate the diff for a set of data and update the visualization without the need for us to tell it which nodes to add or remove (on a per-node or other basis). This PR demonstrates this. Happy to revisit; but this seems to be the way D3 expects us to pass in data (as a single array of objects).